content/version/2/0/code_of_conduct.fr.md

```diff
@@ -0,0 +1,132 @@
+# Charte Code de Conduite Contributeurs
+
+## Notre engagement
+
+En tant que membres, contributeurs et dirigeants, nous nous 
+engageons à rendre la participation dans notre communauté 
+une expérience sans harcelement pour tous, quel que soit l'âge, 
+la physionomie, la présence d'un handicap visible ou invisible, l'ethnie, 
+les caractéristiques sexuelles, l'identité et l'expression de genre, 
+le niveau d'expérience, l'éducation, le statut socio-économique, 
+la nationalité, l'apparence personnelle, la race, la religion, 
+ou l'identité et l'orientation sexuelle.
+
+Nous nous engageons à agir et interagir de manière à contribuer à une communauté 
+ouverte, accueillante, diverse, inclusive et saine.
+
+## Nos critères
+
+Exemples de comportements qui contribuent à créer un environnement positif :
+
+* Faire preuve d'empathie et de bienvaillance envers les autres membres
+* Le respect des différents points de vue, opinions et expériences vécues
+* Donner et accepter poliment les critiques constructives
+* Assumer ses responsabilités et s'excuser auprès des personnes affectées par nos erreurs et apprendre de ces expériences
+* Se concentrer sur ce qui est le meilleur non pas uniquement pour nous en tant qu'individu, mais pour la communauté
+
+Exemples de comportements non acceptables :
+
+* L'utilisation de langage ou d'imageries sexualisés et les avances sexuelles non
+  sollicitées
+* Le _trolling_, les commentaires insultants ou désobligeants et les attaques
+  personnelles ou d'ordre politique
+* Le harcèlement en public ou en privé
+* La publication d'informations privées de tierces personnes, telles que des
+  adresses physiques ou électroniques, sans permission explicite
+* Toute conduite qui pourrait être raisonnablement considérée comme inappropriée
+  dans le milieu professionnel
+
+## Nos responsabilités
+
+Les dirigeants de communauté sont chargés de clarifier et de faire respecter nos normes de
+comportements acceptables et prendront des mesures correctives appropriées et équitables comme
+réponse à tout comportement qu'ils jugent inapproprié, menaçant, offensant ou nuisible.
+
+Les dirigeants de communauté ont le droit et la responsabilité de supprimer, modifier ou rejeter
+les commentaires, les _commits_, le code, les modifications du wiki, les questions et autres contributions qui
+ne sont pas alignés sur ce Code de Conduite et communiqueront les raisons de la modération

```

Comment:
```suggestion
ne sont pas alignés sur ce code de conduite, et communiqueront les raisons des décisions de modération
```

Reply:
I wouldn't capitalize Code of Conduct here. No need, imho, to make this document look more like a legal document.

Reply:
I kept the capital letters in order to respect the English version. I don't know what was the first purpose of the capitalize letters tbh.

Reply:
In any cases, I think that we should change the end of the sentence as you suggested.
But I really don't know if it's interesting to keep the capitals.
```suggestion
ne sont pas alignés sur ce Code de Conduite et communiqueront les raisons de la modération
ne sont pas alignés sur ce Code de Conduite, et communiqueront les raisons des décisions de modération
```